package.json

```diff
@@ -19,23 +19,18 @@
   "license": "MIT",
   "files": ["dist", "src"],
   "source": "src/index.ts",
-  "module": "./dist/index.module.js",
```

Comment:
`"module"` is only used by bundlers, very unlikely anyone will be using this, and if they do, `"exports"` is higher priority anyways in every bundler for the past ~3 years.